packages/codemirror/package.json

```diff
@@ -80,9 +79,9 @@
     "@lumino/widgets": "^1.33.0",
     "react": "^17.0.1",
     "style-mod": "~4.0.0",
-    "y-codemirror.next": "~0.3.0",
```

Comment:
Curious if there is a specific reason for switching from `~` to `^` for that particular package?

Or maybe because it should have been `^` from the beginning?

Reply:
Probably fine either way, was mostly to know the extra context if any.

Reply:
In the JavaScript ecosystem you usually use ^ to get all bugfixes. 

^13.0.0 will use the latest release (13.x.x).

~13.0.0 only accepts patches (13.0.x). So it will never use 13.2.7 (which is a later release with a lot of bugfixes).

In this specific case (0.3.0) it doesn't make a difference.

Reply:
OK then for that particular dependency it makes sense.

The JupyterLab code base sometimes pins on patches for some dependencies like webpack loaders or `typescript`, or for those known to break on minors.

Marking as resolved.